src/com/facebook/buck/cxx/toolchain/objectfile/Machos.java

```diff
@@ -49,6 +50,8 @@
   // Description of object file STAB entries
   static final short N_OSO = (short) 0x66;
 
+  static final String LINKEDIT = "__LINKEDIT";
+
   private Machos() {}
 
   static void setUuid(MappedByteBuffer map, byte[] uuid) throws MachoException {
```

Comment:
you should probably also rename this method to something that is technically more correct - `setUuidIfPresent`